shopify-graphicsmagick.rb

```diff
@@ -0,0 +1,109 @@
+# This file was copied from homebrew/homebrew:graphicsmagick.rb, changes include:
+# * url and sha256 changed to newer tarball
+# * conflict with graphicsmagick added
+# * bottle stanza updated
+class ShopifyGraphicsmagick < Formula
+  desc "Image processing tools collection"
+  homepage "http://www.graphicsmagick.org/"
+  url "http://apt.shopify.com/dist/graphicsmagick/GraphicsMagick-1.4.020150705.tar.bz2"
```

Comment:
https://github.com/Shopify/graphicsmagick/blob/master/Makefile.apt-builder#L17 -- better URL for a tarball?